src/canopy_bioemi_mod.F90

```diff
@@ -52,13 +57,20 @@ SUBROUTINE CANOPY_BIO( ZK, FCLAI, FCH, LAI, FSUN, PPFD_SUN, &
         REAL(RK),    INTENT( IN )       :: TLEAF_SUN(:)    ! Leaf temp for sunlit leaves (K)
         REAL(RK),    INTENT( IN )       :: TLEAF_SHADE(:)  ! Leaf temp for shaded leaves (K)
         REAL(RK),    INTENT( IN )       :: TLEAF_AVE(:)    ! Average Leaf temp (K)
+        REAL(RK),    INTENT( IN )       :: TEMP2           ! Model input 2-m Temperature (K)
         INTEGER,     INTENT( IN )       :: LU_OPT          ! integer for LU type from model mapped to Massman et al. (default = 0/VIIRS)
         INTEGER,     INTENT( IN )       :: VTYPE           ! Grid cell dominant vegetation type
         REAL(RK),    INTENT( IN )       :: MODRES          ! Canopy model input vertical resolution (m)
         REAL(RK),    INTENT( IN )       :: CCE             ! MEGAN Canopy environment coefficient.
         INTEGER,     INTENT( IN )       :: VERT            ! MEGAN vertical integration option (default = 0/no integration)
         INTEGER,     INTENT( IN )       :: CO2OPT          ! Option for co2 inhibition calculation
         REAL(RK),    INTENT( IN )       :: CO2SET          ! User set atmospheric CO2 conc [ppmv]
+
+        INTEGER,    INTENT( IN )       :: LEAFAGEOPT      ! leafage_opt (0= ON, 1= off i.e. GAMMALEAFAGE =1, in canopy_readnml.F90)
+        REAL(RK),    INTENT( IN )       :: PLAI           ! Past LAI [cm2/cm2]
+        REAL(RK),    INTENT( IN )       :: CLAI           ! Current LAI [cm2/cm2]
+        REAL(RK),    INTENT( IN )       :: NDAYS           !Number of days between the past and current LAI
```

Comment:
These variable names could be made more clear? "CLAI" sounds like cumulative LAI. And just looking at "NDAYS" it's not clear what for.